Climbs.py

```diff
@@ -1,18 +1,11 @@
-from PySide6 import QtCore
-from PySide6.QtWidgets import QSizePolicy, QApplication, QMainWindow, \
-QLabel,QPushButton, QToolButton, QVBoxLayout, QWidget, QDialog, QButtonGroup
-import os
-from PySide6.QtUiTools import QUiLoader
-from PySide6.QtCore import QFile, QIODevice
-
+from dataclasses import dataclass   
 
+@dataclass
 class Climb():
-    
-    """Creates a Climb"""
-    def __init__(self, name = None, route = None, grade = None):
-        self.name = name
-        self.route = route
-        self.grade = grade
 
+    name : str = 'unknown'
+    grade : int = 'unknown'
+    route : list[int] = 'un'
```

Comment:
you said its a list of integers but initilised it as a string

Reply:
didn't see the flaw in that when I wrote that but it's because you cannot assign an empty list as its default variable so I should have just assigned None as the default value for everything